docs/editor_manual/new_pages/inserting_images.md

```diff
@@ -1,65 +1,66 @@
-.. _inserting_images:
+(inserting-images)=
```

Comment:
could we please update all refs to be `lower_snake_case` - it appears that the kebab-case here are the exceptions to the norm and it would be good to clean these up while we are doing this work.